### PR TITLE
Fix bad doc links.

### PR DESCRIPTION
**Pull request by geeknoid**
Fix bad links in docs, causing the doc build in istio.io to fail.

Needed for 1.1

### CODE REVIEW

File: mixer/adapter/cloudwatch/config/cloudwatch.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ spec:
...
...IG9mIHRoZSBtZXRyaWMuIE11c3QgYmUgdmFsaWQgY2xvdWR3YXRjaCB1bml0IHZhbHVlLgogW0Nsb3VkV2F0Y2ggZG9jc10oaHR0cHM6Ly9kb2NzLmF3cy5hbWF6b24uY29tL0FtYXpvbkNsb3VkV2F0Y2gvbGF0ZXN0L0FQSVJlZmVyZW5jZS9BUElfTWV0cmljRGF0dW0uaHRtbCkKCg8KBwQAAwECAAQSBFEGTgcKDgoHBAADAQIABhIDUQYKCg4KBwQAAwECAAESA1ELDwoOCgcEAAMBAgADEgNREhMKOwoEBAACAhIDVQQeGi4gVGhlIG5hbWUgb2YgdGhlIGxvZyBncm91cCBpbiBjbG91ZHdhdGNobG9ncy4KCg0KBQQAAgIEEgRVBFIFCgwKBQQAAgIFEgNVBAoKDAoFBAACAgESA1ULGQoMCgUEAAICAxIDVRwdCjwKBAQAAgMSA1gEHxovIFRoZSBuYW1lIG9mIHRoZSBsb2cgc3RyZWFtIGluIGNsb3Vkd2F0Y2hsb2dzLgoKDQoFBAACAwQSBFgEVR4KDAoFBAACAwUSA1gECgoMCgUEAAIDARIDWAsaCgwKBQQAAgMDEgNYHR4KSAoEBAACBBIDWwQiGjsgQSBtYXAgb2YgSXN0aW8gbG9nZW50cnkgbmFtZSB0byBDbG91ZFdhdGNoIGxvZ2VudHJ5IGluZm8uCgoNCgUEAAIEBBIEWwRYHwoMCgUEAAIEBhIDWwQYCgwKBQQAAgQBEgNbGR0KDAoFBAACBAMSA1sgIQoMCgQEAAMDEgRdBWEFCgwKBQQAAwMBEgNdDRQKzAEKBgQAAwMCABIDYAYiGrwBIEEgZ29sYW5nIHRleHQvdGVtcGxhdGUgdGVtcGxhdGUgdGhhdCB3aWxsIGJlIGV4ZWN1dGVkIHRvIGNvbnN0cnVjdCB0aGUgcGF5bG9hZCBmb3IgdGhpcyBsb2cgZW50cnkuCiBJdCB3aWxsIGJlIGdpdmVuIHRoZSBmdWxsIHNldCBvZiB2YXJpYWJsZXMgZm9yIHRoZSBsb2cgdG8gdXNlIHRvIGNvbnN0cnVjdCBpdHMgcmVzdWx0LgoKDwoHBAADAwIABBIEYAZdFQoOCgcEAAMDAgAFEgNgBgwKDgoHBAADAwIAARIDYA0dCg4KBwQAAwMCAAMSA2AgIWIGcHJvdG8z
```
Comment (esnible) [inline mixer/adapter/cloudwatch/config/cloudwatch.yaml]: I am curious what this is and how to decode it.

Comment (geeknoid) [reply mixer/adapter/cloudwatch/config/cloudwatch.yaml]: This is generated content representing the shape of the API. I think it's a serialized proto descriptor.

Comment (esnible) [reply mixer/adapter/cloudwatch/config/cloudwatch.yaml]: When I review a PR containing base64 I like to decode it and make sure there are no passwords.  If I do `kubectl -n istio-system get adapter cloudwatch -o jsonpath='{.spec.config}' | base64 --decode | protoc --decode_raw` I get something.    I looked around and couldn't figure out what to pass to get a type specific decode.

**Comment by geeknoid** [comment]
You're certainly thorough.

@mandarjog @bianpengyuan can provide specific details about what's in the base64 stuff. In the meantime however, it'd be good to merge this change to unblock work in the docs repo for 1.1

Thanks.

**Comment by bianpengyuan** [comment]
@esnible Sorry I am not aware of a way to decode descriptor set to the original proto file, but the base64 content as well as the descriptor set is encoded from the config.proto file in the same directory.

**Pull request merged**